jekyll/_cci2/creating-orbs.md

```diff
@@ -331,3 +331,203 @@ To publish an orb, follow the steps listed below as an org Admin.
 7. Your orb is now published in an immutable form as a production version and can be used safely in builds. You can view the source of your orb by using:
 
 `circleci orb source sandbox/hello-world@0.0.1`
+
+
+## Creating a CircleCI Orb 
+
+This section describes each step of the orb creation and publishing process so you will have a deep understanding of how to write and publish your own orb. This examples enables you to follow the process step-by-step to ensure you write an orb that both adheres to CircleCI requirements while also meeting your own needs.
+
+The following sections describe each step in the orb authoring and publishing process:
+
+1. Meet initial CircleCI prerequisites
+2. Import an existing orb or author your own orb
+3. Validate and publish your orb
+
+## Meet Initial CircleCI Prerequisites
+
+Before you begin creating your own orb, there are a few steps you need to take to ensure that your orb will work with the CircleCI platform and your orb will be properly formatted and structured.
```

Comment:
This paragraph can go. The only reason it should stay is if we choose to add a sub-section on "Settings to enable in the UI", which you don't _need_ in order to author an orb, but you probably want. 

1. Builds-service must be enabled (default is to be ON for all new projects)
2. Orb must be enabled (the security tab disclaimer)

Reply:
added this section, thank you!